### PR TITLE
Add new xmldata BBE to the conversion between XML and record

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

### CODE REVIEW

File: examples/xml-from-record-conversion/xml_from_record_conversion.out [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bal run xml_from_record_conversion.bal
```
Comment (praneesha) [inline examples/xml-from-record-conversion/xml_from_record_conversion.out:1]:
```suggestion
$ bal run xml_from_record_conversion.bal
```

Please check and add the $ sign in all the commands added in all `.out` files.

File: examples/xml-to-record-conversion/xml_to_record_conversion.out [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bal run xml_to_record_conversion.bal
```
Comment (praneesha) [inline examples/xml-to-record-conversion/xml_to_record_conversion.out:1]:
```suggestion
bal run xml_to_record_conversion.bal
```
Same as above.

File: examples/xml-from-record-conversion/xml_from_record_conversion.out (hunk 1 above)
Comment (kalaiyarasiganeshalingam) [reply examples/xml-from-record-conversion/xml_from_record_conversion.out:1]: Do we need to add the $ sign to the master repo BBE?

Comment (praneesha) [reply examples/xml-from-record-conversion/xml_from_record_conversion.out:1]: Sorry, me bad. We need not.

File: examples/xml-to-record-conversion/xml_to_record_conversion.out (hunk 2 above)
Comment (praneesha) [reply examples/xml-to-record-conversion/xml_to_record_conversion.out:1]: Sorry, we need not add the $sign on the master branch as it is available only on the new website.

**Review by praneesha — APPROVED** [review]

**Pull request merged**